locales/en/app.ftl

```diff
@@ -888,26 +894,55 @@ site-nav-ad-callout = Try our other security tools:
 brand-relay = { -brand-relay }
 brand-mozilla-vpn = { -brand-mozilla-vpn }
 
+mobile-menu-label = Main menu
+
+main-nav-button-collapse-label = Collapse menu
+main-nav-button-collapse-tooltip = Collapse menu
+main-nav-button-expand-label = Expand menu
+main-nav-button-expand-tooltip = Expand menu
+main-nav-label = Navigation
+main-nav-link-home-label = Home
+main-nav-link-dashboard-label = Dashboard
+main-nav-link-settings-label = Settings
+main-nav-link-faq-label = FAQs
+main-nav-link-faq-tooltip = Frequently asked questions
+
 ## User menu
 
+# Obsolete
 menu-button-title = User menu
+# Obsolete
 menu-button-alt = Open user menu
+# Obsolete
 menu-list-accessible-label = Account menu
+# Obsolete
 menu-item-fxa-2 = Manage your { -brand-mozilla-account }
-menu-item-fxa-alt-2 = Open { -brand-mozilla-account } page
+# Obsolete
 menu-item-settings = Settings
-menu-item-settings-alt = Open settings page
+# Obsolete
 menu-item-help = Help and support
-menu-item-help-alt = Open help and support page
+# Obsolete
 menu-item-logout = Sign out
 
+user-menu-trigger-label = Open user menu
+user-menu-trigger-tooltip = Profile
+user-menu-manage-fxa-label = Manage your { -brand-mozilla-account }
+user-menu-settings-label = Settings
+user-menu-settings-tooltip = Configure { -brand-fx-monitor }
```

Comment:
```suggestion
user-menu-settings-tooltip = Configure { -brand-mozilla-monitor }
```

Since we have the rebranding in place, when rebasing we should use the new term.

Reply:
d925a2e19